packages/driver/test/cypress/integration/commands/actions/click_spec.coffee

```diff
@@ -131,6 +131,26 @@ describe "src/cy/commands/actions/click", ->
       cy.get("#button").click().then ->
         expect(events).to.deep.eq ["mousedown", "mouseup", "click"]
 
+    describe 'pointer-events:none', ->
+      beforeEach ->
+        cy.$$('<div id="ptr" style="position:absolute;width:200px;height:200px;background-color:aliceblue;"></div>').appendTo cy.$$('#dom')
+        ptrNone = cy.$$('<div id="ptrNone" style="position:absolute;width:400px;height:400px;background-color:salmon;pointer-events:none"></div>').appendTo(cy.$$('#dom'))
+        cy.$$('<div id="ptrNoneChild" style="position:absolute;top:50px;left:50px;width:200px;height:200px;background-color:grey;"></div>').appendTo ptrNone
+      it 'element behind pointer-events:none should still get click', ->
+        cy.get('#ptr').click() # should pass with flying colors
+      it 'should error with message about pointer-events', ->
+        cy.once 'fail', (err) ->
```

Comment:
there is nothing in this test that guarantees the test hit the `fail` callback. for instance, it's theoretically possible for this test to comment out the `fail` callback and still pass.

In order to test that the `fail` callback **did** get called you either spy on it, or you use the mocha `(done)` callback and explicitly call `done()` in the callback.